src/main/java/com/urbanairship/datacube/DataCubeIo.java

```diff
@@ -25,38 +32,38 @@
 /**
  * A DataCube does no IO, it merely returns batches that can be executed. This class wraps
  * around a DataCube and does IO against a storage backend.
- *
+ * <p>
```

Comment:
lol.

Reply:
Intellij autoformat is a cruel mistress. I decided not to go back and revert that, since someone later will just hit the same thing.